packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md

```diff
@@ -1,27 +1,27 @@
 ---
 title: Okta Identity Engine Limitations
-excerpt: The Okta Identity Engine introduces a lot of changes to the Okta platform. Some of these changes result in a lack of support for previously available features.
+excerpt: Okta Identity Engine introduces a lot of changes to the Okta platform. Some of these changes result in a lack of support for previously available features.
 ---
 
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" />
 
-The Okta Identity Engine introduces a lot of changes to the Okta platform. Some of these changes result in a lack of support for previously available features. Additionally, some of these changes result in Okta Identity Engine features not supported for use with Okta Classic APIs.
+Okta Identity Engine introduces a lot of changes to the Okta platform. Some of these changes result in a lack of support for previously available features. Additionally, some of these changes result in the Identity Engine features not supported for use with Okta Classic Engine APIs.
 
-> **Note:** This doc is designed for people who are familiar with Okta Classic. If you are new to Okta and Okta Identity Engine, see [Get started](https://help.okta.com/en/oie/okta_help_CSH.htm#ext-get-started-oie) with Okta Identity Engine.
+> **Note:** This doc is designed for people who are familiar with the Classic Engine. If you are new to Okta and the Identity Engine, see [Get started](https://help.okta.com/en/oie/okta_help_CSH.htm#ext-get-started-oie) with the Identity Engine.
 
-Are you an admin? See the Okta Identity Engine [Limitations](https://help.okta.com/en/oie/okta_help_CSH.htm#ext-oie-limitations) doc for admins.
+Are you an admin? See the Identity Engine [Limitations](https://help.okta.com/en/oie/okta_help_CSH.htm#ext-oie-limitations) doc for admins.
 
-### Okta Classic features not supported in Okta Identity Engine
+### Classic Engine features not supported in the Identity Engine
 
 #### Event Type availability for Event Hooks
 
-**What Changed:** The following Event Types aren't available in the Okta Identity Engine because Device Trust isn't currently supported:
+**What Changed:** The following Event Types aren't available in the Identity Engine because Device Trust isn't currently supported:
 
 * `user.authentication.authenticate`
 * `user.credential.enroll`
 
-The following Event Types are available only in the Okta Identity Engine and can't be used by Okta Classic customers:
+The following Event Types are available only in the Identity Engine and can't be used by Classic Engine customers:
```

Comment:
Given the number of "Identity Engine" and "Classic Engine", I'm thinking that the word "Okta" preceding them has been taken out. Please do a search and make sure there's consistency.